fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -735,6 +912,20 @@ public long loadBackends(DataInputStream dis, long checksum) throws IOException
         return checksum;
     }
 
+    public void loadComputeNodes(DataInputStream dis) throws IOException {
```

Comment:
You should make this part of image compatible. So you should make this part to a json object. Refer to CatalogMgr.loadCatalogs.

Reply:
In addition, you should add some message to checksum.